src/patternfly/sass-utilities/scss-variables.scss

```diff
@@ -1,4 +1,5 @@
 @import './init'; // initialize versioning
+$pf-v5-global--placeholders-loaded: false;
```

Comment:
Thinking about this a little more, wdyt about localizing everything just to the placeholders file? Seems a little odd for this to be with the rest of our other global vars. I dropped this in place, and it seems to work. 

```diff
diff --git a/src/patternfly/sass-utilities/themes/dark/placeholders.scss b/src/patternfly/sass-utilities/themes/dark/placeholders.scss
index 1d3f1307..10ee60de 100644
--- a/src/patternfly/sass-utilities/themes/dark/placeholders.scss
+++ b/src/patternfly/sass-utilities/themes/dark/placeholders.scss
@@ -1,5 +1,12 @@
-%pf-v5-theme-dark--t-dark {
-  .#{$button} {
-    --#{$button}--m-primary--BackgroundColor: var(--#{$pf-global}--primary-color--300);
+$pf-v5-global--placeholder-loaded: false !default;
+
+// Prevents loading this file multiple times in a single compilation
+@if not $pf-v5-global--placeholder-loaded {
+  %pf-v5-theme-dark--t-dark {
+    .#{$button} {
+      --#{$button}--m-primary--BackgroundColor: var(--#{$pf-global}--primary-color--300);
+    }
   }
 }
+
+$pf-v5-global--placeholder-loaded: true;
```